### DISCUSSION TITLE
`/usr/lib/ruby/gems/3.0.0/cache`. It is likely that you need to grant write permissions for that path.

### DISCUSSION THREAD

**Discussion by minsoehan** [ruby/rubygems#6503] [Q&A] [answered]
I am trying to execute `$ bundle install` in jekyll-generated folder, in Arch Linux. but I am having the following issue.
```
Bundler::PermissionError: There was an error while trying to write to `/usr/lib/ruby/gems/3.0.0/cache`. It is likely
that you need to grant write permissions for that path.
```
what I am missing.

**Comment by guss77** [comment, marked as answer]
If you want to install the bundle locally, it is recommended to first run `bundle config --local path .bundle` (or some other path instead of `.bundle`). Bundler used to say that when you try to install things as a user without setting up a local configuration - but for some reason it doesn't do that anymore.

**Comment by erwin** [comment]
I was having the same issue on `arch`. 

Your suggested setting of `--local path` fixed it for me!

```
bundle config --local path .bundle
```

Thank you @guss77 !
